WORKING-GROUPS.md

```diff
@@ -155,7 +155,7 @@ Build, test, release
 Artifact | Link
 ---|---
 Forum | [istio-test-release@](https://groups.google.com/forum/#!forum/istio-test-release)
-Community Meeting VC | [Thursdays at 13:00 PT - Weekly](https://plus.google.com/hangouts/_/google.com/istio-test)
+Community Meeting VC | [Monday at 10:00 PT - Weekly](https://plus.google.com/hangouts/_/google.com/istio-test)
 Community Meeting Calendar | [Calendar Invitation](https://calendar.google.com/event?action=TEMPLATE&tmeid=YjkwZGxpaGFpajJnMDZpb2w0a3Qxa2kwdmdfMjAxNzExMzBUMjEwMDAwWiBoa2xhaUBnb29nbGUuY29t&tmsrc=hklai%40google.com)
```

Comment:
this still points to Thu 1pm

Reply:
Thanks. The old one was actually a past instance, and was not even recurring. =)